crd/sparkcluster.crd.yaml

```diff
@@ -35,6 +35,28 @@ spec:
                             type: string
                           instances:
                             type: integer
+                          masterPort:
+                            type: integer
```

Comment:
If you want to you could add optional minimums here. It shouldn't be less than 0 or 1 I assume
some for the other ports

Reply:
Yes the question is: Do we catch wrong inputs already in the crd or just via product config?
For ports e.g. if we run the operators as user you cannot pick anything lower than 1024. With root you have the full range. We cannot express that in the crd as far as i know? But then why would run them as root. Just an example.

Reply:
Yeah you're right but it doesn't hurt to catch things twice. And this creates some auto-generated docs.
Fine for now.